src/styles/_fonts.scss

```diff
@@ -1,6 +1,17 @@
 @font-face {
-  font-family: "Roboto";
-  src: url("../fonts/Roboto-Regular-webfont.woff") format("woff");
+  font-family: "Avenir-Book";
```

Comment:
Adding fonts. To use 2 font-weights per font, you must specify the font source, font name, and font size.
```
@font-face {
  font-family:  Avenir;
  src: url(/fonts/Avenir-Book.ttf);
  font-weight: 300;
}

@font-face {
  font-family:  Avenir;
  src: url(/fonts/Avenir-Heavy.ttf);
  font-weight: 900;
}
```
Next, we can use one Avenir font everywhere and have 2 of its 300 and 900 font-weights.